src/status_im/contexts/wallet/events.cljs

```diff
@@ -332,6 +332,18 @@
    {:fx [[:dispatch [:hide-bottom-sheet]]
          [:dispatch [:browser.ui/open-url (str explorer-link "/" address)]]]}))
 
+(rf/reg-event-fx :wallet/reload
+ (fn [_]
+   {:fx [[:dispatch-n [[:wallet/get-wallet-token]]]]}))
+
+(rf/reg-event-fx :wallet/start-wallet
+ (fn [_]
+   {:fx [[:json-rpc/call
+          [{:method   "wallet_startWallet"
+            :on-error #(log/info "failed to start wallet"
+                                 {:error %
+                                  :event :wallet/start-wallet})}]]]}))
+
 (rf/reg-event-fx :wallet/initialize
  (fn []
-   {:fx [[:dispatch-n [[:wallet/get-ethereum-chains] [:wallet/get-accounts]]]]}))
+   {:fx [[:dispatch-n [[:wallet/start-wallet] [:wallet/get-ethereum-chains] [:wallet/get-accounts]]]]}))
```

Comment:
Since we are using `:fx` I think we should only use two `:dispatch`  instead of a `:dispatch-n`.

As far as I remember `dispatch-n`  existed before re-frame introduced `:fx`, and now that we are using `:fx`, mixing it with `:dispatch-n` seems not too natural

Reply:
Yes, @ulisesmac. `:fx` will dispatch events in order. I will update this 👍 
Thanks!